src/client/input-leapc.cpp

```diff
@@ -44,6 +44,22 @@ int client_main(int argc, char** argv)
     Log log;
     EventQueue events;
 
+    // TODO: Remove once Wayland support is stabilised.
+
+    // This block only warns when `libportal` and `libeis` aren't available - as well as if the top-level CMake option is enabled - by default, it is.
```

Comment:
Please keep lines to 100 chars or less.

Reply:
Also move this check to separate function.

Reply:
Also did the same for the server. We don't need to do this for the daemon - that's Win32. See: fe1dd285